brave/src/main/java/brave/TracingCustomizer.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2013-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave;
+
+import brave.handler.FinishedSpanHandler;
+import zipkin2.reporter.Reporter;
+
+/**
+ * This allows configuration plugins to collaborate on building an instance of {@link Tracing}.
+ *
+ * <p>Notably, this decouples framework-level configuration (such as {@link
```

Comment:
It's not so obvious how these decouple the configuration. Maybe `this can be used to decouple blah and blah by having each layer create its own customizer and applying them all to construct the final Tracing`

Reply:
sgtm!

Reply:
I think I might add a code example, as I'm struggling with words that don't also repeat the first sentence. Thanks for helping me think through this.

Reply:
I changed the wording instead. lemme know if this works.